### DISCUSSION TITLE
Fresh installation of Hassio VM on proxmox not work

### DISCUSSION THREAD

**Discussion by twiti7** [tteck/Proxmox#1318] [Help Needed]
I install fresh latest proxmox and used latest hassio 9.5 and it want boot

**Comment by tteck** [comment]
Did you ensure that UEFI Boot & Virtualization is enabled and Secure Boot is disabled in your machine’s BIOS?

**Comment by twiti7** [comment]
yes virtualization is enabled UEFI too and secure boot disabled

**Comment by tteck** [comment]
Tell me about your machine specs.

**Comment by twiti7** [comment]
It is DELL WYSE ZX0 Z90D7 1.65GHZ  8192 MB of RAM and 128 GB SSD. Everything was fine until I mage update of proxmox. So I try made fresh install and get all stuff from backup but now I can't start even Hassio.
It looks like here https://github.com/home-assistant/operating-system/issues/2357

**Reply by tteck** [reply]
And it worked fine before the Proxmox update?

**Reply by twiti7** [reply]
I installed a proxmox older proxmox 7.3-3 and it works fine I mean HA was installed with your script. With a latest version of proxmox installation process also finished well, but container stucks - bootloop.

**Reply by tteck** [reply]
Try the older `pve-edk2-firmware` that 7.3 uses.
Run this command in the Proxmox VE 7.4 shell
```
apt-get install pve-edk2-firmware=3.20220526-1
```
Reboot Proxmox
